.github/workflows/charts-release.yaml

```diff
@@ -0,0 +1,28 @@
+name: Charts-Release-Validations
+
+on:
+  pull_request:
+    branches:
+      - release-v*
+
+jobs:
+  build:
+    name: Checkpoint 0 - release.yaml validation
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout base branch
+        uses: actions/checkout@v4
+
+      - name: Checkout PR
+        run: gh pr checkout ${{ github.event.pull_request.number }}
+        env:
+            GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Checkout into branch
+        run: git checkout -b staging-pr-workflow
+
+      - name: Pull scripts
+        run: sudo make pull-scripts
+
+      - name: Validate Charts Release
+        run: sudo make validate-release-charts BRANCH=${{ github.event.pull_request.base.ref }} GH_TOKEN=${{ secrets.GITHUB_TOKEN }} PR_NUMBER=${{ github.event.pull_request.number }}
```

Comment:
Does these commands really need to use `sudo`? Maybe you can drop it by updating permissions on files

Reply:
I am going to test this now.
Good suggestion